maven-release-manager/src/main/java/org/apache/maven/shared/release/DefaultReleaseManager.java

```diff
@@ -592,8 +586,6 @@ protected void clean( AbstractReleaseRequest releaseRequest  ) throws ReleaseFai
     @Override
     public void clean( ReleaseCleanRequest cleanRequest ) throws ReleaseFailureException
     {
-        updateListener( cleanRequest.getReleaseManagerListener(), "cleanup", PHASE_START );
```

Comment:
Why is this information not necessary anymore?

Reply:
same "non phase" message that creates confusion when it looks as a phase

Reply:
Agreed.